bootstrap/azure/util.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Use the config file specified in $APOLLO_CONFIG_FILE, or default to
+# config-default.sh.
+function set_vpn() {
+  while true; do
+  read -p "Do you want to start the VPN and setup a connection now (y/n)?" yn
+    case $yn in
+        [Yy]* ) ovpn_start;ovpn_client_config; break;;
+        [Nn]* ) exit;;
+        * ) echo "Please answer y or n.";;
+    esac
+  done
+}
+
+ansible_ssh_config() {
+  pushd "${APOLLO_ROOT}/terraform/${APOLLO_PROVIDER}"
+    export APOLLO_bastion_ip=$( terraform output bastion.ip )
+
+    # Virtual private cloud CIDR IP.
+    ip=$( terraform output vn_cidr_block.ip )
+    export APOLLO_network_identifier=$( get_network_identifier "${ip}" )
+
+    cat <<EOF > ssh.config
+  Host bastion $APOLLO_bastion_ip
+    StrictHostKeyChecking  no
+    User                   ubuntu
+    HostName               $APOLLO_bastion_ip
+    ProxyCommand           none
+    IdentityFile           $TF_VAR_private_key_file
+    BatchMode              yes
+    PasswordAuthentication no
+    UserKnownHostsFile     /dev/null
+
+  Host $APOLLO_network_identifier.*
+    StrictHostKeyChecking  no
+    ServerAliveInterval    120
+    TCPKeepAlive           yes
+    ProxyCommand           ssh -q -A -F $(pwd)/ssh.config ubuntu@$APOLLO_bastion_ip nc %h %p
+    ControlMaster          auto
+    ControlPath            ~/.ssh/mux-%r@%h:%p
+    ControlPersist         30m
+    User                   ubuntu
+    IdentityFile           $TF_VAR_private_key_file
+    UserKnownHostsFile     /dev/null
+EOF
+  popd
+}
+
+ssh_thumbprint() {
+  pushd "${APOLLO_ROOT}/terraform/${APOLLO_PROVIDER}"
+    if [ ! -f "ssh_thumbprint" ]; then
+      touch ssh_thumbprint
+    fi
+  popd
+}
+
+apollo_down() {
+  pushd "${APOLLO_ROOT}/terraform/${APOLLO_PROVIDER}"
+    terraform destroy -var "azure_settings_file=${TF_VAR_azure_settings_file}" \
+      -var "region=${TF_VAR_region}"
+  popd
+}
+
+ovpn_start() {
+  pushd "${APOLLO_ROOT}/terraform/${APOLLO_PROVIDER}"
+    echo "... initialising VPN setup" >&2
+    /bin/sh -x bin/ovpn-init
+    /bin/sh -x bin/ovpn-start
+  popd
+}
+
+ovpn_client_config() {
```

Comment:
@ravbaba  can you raise a issue to do this. we can then look to merge this and iterate for improvements